stdlib/Makefile

```diff
@@ -244,18 +244,14 @@ GNUISH_SED = \
   $(if $(filter X,$(shell echo x | $(SED) -E -e 's/./\u&/' 2>/dev/null)),\
        $(SED),$(error GNU sed is needed for make depend))
 
-.INTERMEDIATE: .depend.tmp
-
-.depend.tmp:
-	$(OCAMLDEP_CMD) $(filter-out stdlib.%,$(wildcard *.mli *.ml)) > $@
-	$(OCAMLDEP_CMD) -pp "$(AWK) -f ./remove_module_aliases.awk" \
-	  stdlib.ml stdlib.mli >> $@
-
 .PHONY: depend
-depend: .depend
-
-.depend: .depend.tmp
+depend:
```

Comment:
This is correct as it is; I'd suggest one of two possible changes:

1. Keep the `depend: .depend` dependency, as this would allow `.depend.tmp` and `.depend` to be written as `$@.tmp` and `$@` in the recipe, which I think is actually slightly clearer
2. We could switch to a single shell command and use `{ ... }`, as @gasche alludes to:
```make
depend:
	{ \
	  $(OCAMLDEP_CMD) $(filter-out stdlib.%,$(wildcard *.mli *.ml)) ; \
	  $(OCAMLDEP_CMD) -pp "$(AWK) -f ./remove_module_aliases.awk" stdlib.ml stdlib.mli ; \
	} | $(GNUISH_SED) -E ... > .$@
```